animation_nodes/nodes/mesh/get_custom_attribute.py

```diff
@@ -0,0 +1,49 @@
+import bpy
+from bpy.props import *
+from ... base_types import AnimationNode
+from ... data_structures import DoubleList
+
+class GetCustomAttributeNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GetCustomAttributeNode"
+    bl_label = "Get Custom Attribute"
+    errorHandlingType = "EXCEPTION"
+
+    dataType: StringProperty(name = "Color Mode", default = "FLOAT", update = AnimationNode.refresh)
```

Comment:
What is this property exactly? It is not drawn, it is not assigned, it is named `Color Mode`. I am not sure what it does.

Reply:
Sorry, the name is wrong. Actually, this is added to change the output type according to attribute data. Is there any other way to do this?

Reply:
@3DSinghVFX This is probably not a good idea. What if this node is inside a group and the attribute changes?
I would say the best solution is to make this explicit. Add an enum for the output type. If the attribute does not match the output type, warn the user to change the enum manually.

While at it, `getAllAttributes` seems like it returns a list. Just remove it and use `mesh.attributes` directly.

Reply:
> While at it, `getAllAttributes` seems like it returns a list. Just remove it and use `mesh.attributes` directly.

Omar, this gave an error:
```
attributes = mesh.attributes
AttributeError: 'animation_nodes.data_structures.meshes.mesh_data.M' object has no attribute 'attributes'
```

Reply:
@3DSinghVFX  Probably because it is cdef-ed. In this case, just rename it to `getAttributes`.

Reply:
We have already the `getAttributes` method which gave attributes by type. That is why I added this function.

Reply:
@3DSinghVFX How about renaming the other one `getAllAttributesOfType`?
I suppose we can remove this function altogether and add another one `getAttributeByName` to hide the underlying dict.

Reply:
Okay. We all attribute names to show in the `Get Custom Attributes` node.